lib/membrane_ice_plugin/ice_endpoint.ex

```diff
@@ -212,6 +219,13 @@ defmodule Membrane.ICE.Endpoint do
             parent: self()
           )
 
+        {:ok, _pid} =
+          Membrane.ICE.TURNCleaner.start_under(
+            state.turn_cleaner_sup,
+            pid: self(),
+            turn: udp_integrated_turn
+          )
+
```

Comment:
Maybe move starting turn and starting cleaner to separate private function, to avoid repeating this same code in lines 265-271?

Reply:
In the first version, starting cleaner was in `Utils.start_integrated_turn_servers` but we decided to move it out of this function to separate both actions. Introducing a private function sounds like a return to the first version.